Hw1/Hw1/BWT.cs

```diff
@@ -0,0 +1,98 @@
+﻿// <copyright file="BWT.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Hw1
+{
```

Comment:
Используйте file-scoped namespaces
```suggestion
namespace Hw1;
```